core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -662,7 +665,8 @@ private[deploy] class Master(
 
     state = RecoveryState.ALIVE
     schedule()
-    logInfo("Recovery complete - resuming operations!")
+    val timeTakenNs = System.nanoTime() - recoveryStartTimeNs
+    logInfo(s"Recovery complete in ${timeTakenNs / 1000000000d}s - resuming operations!")
```

Comment:
`0.043445167s` may be less readable for humans. How about rounding to milliseconds？

`TimeUnit.NANOSECONDS.toMillis(timeTakenNs) ms` or `TimeUnit.NANOSECONDS.toMillis(timeTakenNs) / 1000.0 s`

Reply:
Sure!